lib/cutest.rb

```diff
@@ -171,6 +171,14 @@ def assert_raise(expected = Exception)
       success
     end
   end
+  
+  # Assert that the block raises no exceptions.
+  def assert_nothing_raised
+    yield
+    success
+  rescue Exception => exception
```

Comment:
This is a good point. Fortunately, it is consistent with the behaviour of cutest's `assert_raise`.

I don't know if this behaviour is desirable. It's generally bad practice to default to rescuing any exception. Perhaps `assert_raise` should only rescue `StandardError` and up by default and require an explicit exception class to rescue lower level exceptions (which https://github.com/djanowski/cutest/pull/21 allows for).